cumulusci/tasks/bulkdata/generate_from_yaml.py

```diff
@@ -27,6 +27,9 @@ class GenerateDataFromYaml(BaseGenerateDataTask):
         "vars": {
             "description": "Pass values to override options in the format VAR1:foo,VAR2:bar"
         },
+        "plugin_options": {
+            "description": "Pass values to override plugin options in the format VAR1:foo,VAR2:bar"
```

Comment:
Don't you need to run this through process_list_of_pairs_dict_arg in order to support that format on the CLI in addition to mappings in yaml?

Reply:
Good point. I was only thinking of this class as an implementation detail of the other one, but of course it is its own task that people can and do use. Fixed in 5e7182fa2